src/services

```diff
@@ -724,17 +724,45 @@ p({
   "name": "ruleDAO",
   "serve": true,
   "serviceScript": """
+    dao = new foam.dao.EasyDAO.Builder(x)
+      .setEnableInterfaceDecorators(false)
+      .setOf(foam.nanos.ruler.Rule.getOwnClassInfo())
+      .setRuler(false)
+      .setInnerDAO(x.get("localRuleDAO"))
+      .build();
+
+    dao.addPropertyIndex(new foam.core.PropertyInfo[] { foam.nanos.ruler.Rule.NAME });
+    dao = new foam.dao.ValidatingDAO(x, dao);
+
+    return dao;
+  """,
+  "client": """
+    {
+      "of": "foam.nanos.ruler.Rule"
+    }
+  """
+})
+
+p({
+  "class": "foam.nanos.boot.NSpec",
+  "name": "localRuleDAO",
+  "serve": true,
```

Comment:
local daos should not be served.

Reply:
fixed changed some other local daos to serve false too which were previously serve true